pandas/tools/tests/test_tile.py

```diff
@@ -288,7 +288,10 @@ def test_qcut_duplicates_bin(self):
 
     def test_single_bin(self):
         # issue 14652
-        expected = Series([0, 0])
+        # Explicit dtype since Series produces int64 for ints, while cut
+        # (due to numpy.searchsorted) would use int32 on i386, so let's assure
+        # correct default to the architecture int
```

Comment:
we don't test on 32-bit, but if this works that is ok

specify ``dtype='intp'`` here